### ISSUE TITLE
Correct terrainPlains and Polar references

### ISSUE THREAD

**Issue by Zeno410** [Team-RTG/Realistic-Terrain-Generation#578]
The terrainPlains code generates dunes, which is not appropriate with most plains. The ones that use dunes need to be converted to terrainPolar (I made a renamed version) and the existing terrainPlains rewritten to not produce dunes. In addition, the existing terrainPolar users should probably be converted to the new one as the old one has very limited parameters.

**Comment by whichonespink44** [comment]
Is it safe to close this one?

**Comment by Zeno410** [comment]
Up to you. I think all the plains have been fixed but I haven't checked. There's still a lot of references to the over-standardized terrainPolar.

**Comment by whichonespink44** [comment]
I'll leave it open then as I haven't checked them all yet. I'll keep this in mind though as I'm going through all of the biome re-decorations.

**Comment by whichonespink44** [comment]
All the Plains biomes are now using terrainPlains (there was only one that I had to fix).

I attempted to do some work on terrainPolar, but ended up reverting it all. The standardized polar biomes are fine for now, as they're spread across different mods so won't typically appear together. However, if anyone else wants to re-design the polar biomes, go for it - as long as at least one of them has massive dunes! :stuck_out_tongue:
